org.opent2t.sample.binaryswitch.superpopular/org.opent2t.sample.binaryswitch.superpopular.raml

```diff
@@ -188,3 +193,24 @@ traits:
         body:
           application/json:
           example: {}
+
+/?$availability: |
```

Comment:
Moved! Please review.

---
In reply to: [105240838](https://github.com/openT2T/translators/pull/197#discussion_r105240838) [](ancestors = 105240838)